TAF/BDT/Steps/JobListingTestsSteps.cs

```diff
@@ -0,0 +1,56 @@
+﻿using NUnit.Framework;
+using TAF.Core.Browser;
+using TAF.Core.Utilities;
+using TAF.Web.Pages;
+
+namespace TAF.Tests.BDT.Steps
+{
+    [Binding]
+
+    public class JobListingTestsSteps
+    {
+        private MainPage _mainPage;
+        private JobListingsPage _jobListingsPage;
+        private string defaultSearchMessage;
+
+        [Given(@"setup page objects for job listing tests")]
+        public void GivenSetupPageObjectsForJobListingTests()
```

Comment:
It is not a good practice to create initialization steps. It is better to initialize pages using => symbol before all steps. Best practice here is 1 page to 1 steps class